datafusion/common/src/scalar/mod.rs

```diff
@@ -1715,6 +1715,22 @@ impl ScalarValue {
                 )?;
                 Arc::new(array)
             }
+            DataType::Utf8View => {
```

Comment:
Maybe just `build_array_string!(StringViewArray, Utf8View)`? 
Can you also move this case closer to where we handle `Utf8` and `LargeUtf8`?